test.js

```diff
@@ -9,12 +10,22 @@ test('tempWrite(string)', async t => {
 	t.is(path.basename(filepath), 'test.png');
 });
 
+test('tempWrite(stream)', async t => {
+	const readable = new stream.Readable({
+		read() { /* noop */ }
+	});
```

Comment:
[Docs](https://nodejs.org/api/stream.html#stream_implementing_a_readable_stream) say:

 > ... must implement the readable._read() method

And [this person](http://stackoverflow.com/a/22085851/2990144) says not implementing it will cause a crash in the REPL. Although, I didn't try it honestly. Was just being paranoid.